bootstrap/local.js

```diff
@@ -8,7 +7,10 @@ import { printRoute } from '../src/helpers';
 import DynamoLogger from '../src/Logging/DynamoLogger';
 
 // Configure to use DynamoDB Local in development:
-AWS.config.update({ region: 'us-east-1', endpoint: 'http://localhost:45670' });
+dynamoose.aws.sdk.config.update({
```

Comment:
I'm not entirely sure, but I believe that Dynamoose was keeping it's own instance of the SDK & so this `AWS.configure` call was not taking effect. By using this `dynamoose.aws.sdk.config.update` (whew) method, we're able to make sure we configure the instance that Dynamoose is using.